### PR TITLE
Working page

### PR DESCRIPTION
**Pull request by rloterh**
- Positioned elements using CSS3 grids and flexbox
- Embedded images
- Worked with forms (buttons)
- Applied the box model

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,596 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
+<!--<![endif]-->
```
Comment (HADDADSOHAIB) [inline index.html]: Commented code here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+  <title>NYT Clone</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+  <!--[if lt IE 7]>
+      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
```
Comment (HADDADSOHAIB) [inline index.html]: Commented code here

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @rloterh  and @eypsrcnuygr  :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, there are still some changes to get the project approved:
- Please fix linter errors:
Screenshot from 2020-06-15 13-34-26
- You have commented code in your project, please delete that.
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Please fix the validators errors and warnings:
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
Screenshot from 2020-06-15 13-39-44
- The logo in the navbar must be a link:
- [ ] The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
- The navbar is not similar to the original:
The original:
Screenshot from 2020-06-15 10-38-49
 Your Project:
Screenshot from 2020-06-15 10-39-36
- The social media icons should be links, both on the beginning and on the end of the articles:
Screenshot from 2020-06-15 13-46-01
- Editors picks articles should be links as the original:

Screenshot from 2020-06-15 13-47-15

- There is a sticky footer on the original, please add that on your project:
Screenshot from 2020-06-15 10-49-53
- I think the proportion of the paragraphs in page layout are off a lot, try to adjust that:
your project:
Screenshot from 2020-06-15 13-48-06
original:
Screenshot from 2020-06-15 13-49-50
- Please fix the image of the right:
Screenshot from 2020-06-15 13-50-11
- The footer is mess up, please fix that, also the article on the footer should be link, all of them, including the images:
Screenshot from 2020-06-15 13-51-38

Screenshot from 2020-06-15 13-51-46
Screenshot from 2020-06-15 13-51-52

### Once you are done with the changes, please asked for another code review. :wave:

**Comment by sumancrest0001** [comment]
**Status: changes required
Hello TEAM.** thank you for making most of the changes requested by the previous TSE. Your project and repository are looking much more professional and clean. But there are still some issues that need to be addressed to get approved. Let's have a look at them.
**[NOTE] All the changes are mandatory.**

1. As stated by the previous TSE, the proportion of the page is different from the original page. Please change it which will give your project a similar look to the original one.

2. Most popular section layout is different from the original page. Please compare them
your :point_down: 
Screenshot from 2020-06-16 17-10-55

Original one :point_down: 
Screenshot from 2020-06-16 17-11-11

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+  width: 16px;
+  height: 16px;
+  padding-right: 20px;
+}
+
+#search-icon {
+  width: 16px;
+  height: 16px;
+  padding-right: 20px;
+}
```
Comment (sumancrest0001) [inline css/styles.css]: We have to remove repeated stylings and follow the DRY principle. Please remove all the repeated codes.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the requested changes and resubmit for another code review.**
If you have any questions regarding the code review, DM me on slack **Suman Shrestha**

**Happy coding :+1:**

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Great job implementing the changes 👍. I see that you have made some solid changes. Good job!!

However, we need to work on some changes before we can merge.

- The `inflation-container` is overflowing. I see that you have applied `overflow-x: hidden;` to the body. It's not a solution, it's more like a band-aid. Fix this.

Annotation 2020-06-16 221847

### [OPTIONAL]

- The social icons need to be centered.
Annotation 2020-06-16 224040

- Increase the width of `article-content`

Apply these changes and request another code review.

Happy coding 👨‍💻

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+.main-img {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.inflation-description span {
+  font-weight: 700;
+  font-size: 11;
+}
```
Comment (peterrobert) [inline css/styles.css]: Font-size is a unit. You need to add px for pixels

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,535 @@
...
+
+.section-2 aside div {
+  display: flex;
+  margin-bottom: 15px;
+}
+
+.inflation-desc p {
+  font-size: 16px;
+  color: 3333;
+}
```
Comment (peterrobert) [inline css/styles.css]: You have an extra three on the color.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
**HELLO** @rloterh :wave: 
**Great work so far. You have implemented the requested changes by the previous TSE**
However, there are a few more changes to be made so that we can approve this project:

- [ ] Kindly improve on your footer section. It doesn't  look like the original
**yours**
nyt_footer

**original**
nyt_footer_original

- [ ] Kindly also improve on this section. It needs to look like the original.

**yours**
sec
**original**
sec-original

Also, there are some errors that need fixing in your CSS check below :arrow_down: 

**Kindly submit another code review when you fix all the suggestions**

### HAPPY CODING

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
There are still some changes to be made in your project. Kindly you haven't made the previous changes requested. You need to make your webpage look like the original. :smile: 

- [ ] Also kindly try to improve this section.
fill
 Kindly if you compare it with the original yours is not well aligned.

**once you make those changes kindly submit your work for another review**

If you have any questions concerning the review you can contact me on @peterrobert 

### HAPPY CODING :muscle:

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @rloterh and @eypsrcnuygr    :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, the proportion of your page elements are still off, a lot, learning how to size and place an element is a necessary skill and that the goal of this project, please take the time to master that, you will need it later. I will attach some screen shot to show how your project appears in my browser:
Screenshot from 2020-06-17 14-09-12
This part is great.
Screenshot from 2020-06-17 14-09-55
The width of the paragraph is tiny.
Screenshot from 2020-06-17 14-10-36
This also very small.
Screenshot from 2020-06-17 14-11-17
[Optional]This is great, but you can improve by positioning the element in the center, it is not well centered. 
Screenshot from 2020-06-17 14-13-34
The width of the article should be the same as the image, so make the article larger then that, you can keep the image on the same size or make it just a little smaller. Follow the original project layout.
Screenshot from 2020-06-17 14-15-44
This part should take the full width.

### once you are done with the changes, please ask for another code review. :+1:

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job! :+1:
- You can now merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Comment by rloterh** [comment]
Resolved issues

**Comment by rloterh** [comment]
Resolved Issues

**Pull request merged**